buildSrc/src/main/groovy/org/apache/beam/gradle/GrpcVendoring_1_43_2.groovy

```diff
@@ -0,0 +1,196 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.beam.gradle
+
+/**
+ * Utilities for working with our vendored version of gRPC.
+ */
+class GrpcVendoring_1_43_2 {
+
+  static def guava_version = "30.1.1-jre"
+  static def protobuf_version = "3.19.2"
+  static def grpc_version = "1.43.2"
+  static def gson_version = "2.8.9"
+  // tcnative version from https://github.com/grpc/grpc-java/blob/master/SECURITY.md#netty
+  static def netty_version = "4.1.63.Final"
+  static def netty_ssl_version = "2.0.38.Final"
+  // google-auth-library version from https://search.maven.org/artifact/io.grpc/grpc-auth/1.43.2/jar
+  static def google_auth_version = "0.22.2"
+  // proto-google-common-protos version from https://search.maven.org/artifact/io.grpc/grpc-protobuf/1.43.2/jar
+  static def proto_google_common_protos_version = "2.0.1"
+  static def opencensus_version = "0.28.0"
```

Comment:
Would you add source code comment on how you get this version? I guess it's from 

https://search.maven.org/artifact/io.grpc/grpc-census/1.43.2/jar